### PR TITLE
Initial Timbre integration

### LINKED ISSUE #12: Origin of log entries misreported
I just got this in my logs:
`16-Jan-14 00:30:13 rufus-laptop.home DEBUG [slf4j-timbre.adapter] -  Entry count for : https://api.twitter.com:443 : 0`

The real origin is [netty](https://github.com/AsyncHttpClient/async-http-client/blob/e7798c8926ad020d3b1f845c44f0e12255fb02c0/client/src/main/java/org/asynchttpclient/netty/channel/DefaultChannelPool.java#L197) but it is being misreported as coming from `slf4j-timbre.adapter` itself.

Need to investigate why

### PR DESCRIPTION
**Pull request by rufoa**
- populate Timbre `:?ns-str` with slf4j logger name (the FQN of the log site by convention)
- populate Timbre `?:file` and `?:line` from call stack
- put slf4j Marker name, if present, into Timbre `:context` map with key `:marker`

closes #12

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-  :profiles {:dev {:dependencies [[midje "1.8.3"]]
-                   :plugins [[lein-midje "3.2"]]}}
-
-  :aot :all
-
-  :scm {:name "git"
-        :url "https://github.com/fzakaria/slf4j-timbre"}
-
-  )
+(defproject com.fzakaria/slf4j-timbre "0.3.1"
```
Comment (fzakaria) [inline project.clj]: why does it highlight the whole file here

**Comment by fzakaria** [comment]
Looks good to me.
I would recommend 1 test to validate the line / namespace.
Just search/replace that the log statement timbre reports has the test namespace ?

**Comment by rufoa** [comment]
yeah I've a few tests to add yet

**Comment by rufoa** [comment]
Improved the existing tests and added the new ones

https://github.com/fzakaria/slf4j-timbre/blob/46fa4423c569d17426ecbc45da26009e9e43f066/test/slf4j_timbre/t_adapter.clj

**Pull request merged**